python/test/test_env.py

```diff
@@ -0,0 +1,48 @@
+import os
+import re
+
+import pytest
+from timeplus.env import Environment
+WORKSPACE_LEN = 8
+APIKEY_LEN = 60
+
+@pytest.fixture()
+def test_env():
```

Comment:
this function is not required, as you can re-use the env defined in the fixture of conftest